pages/api/candidateApiClientModule.ts

```diff
@@ -0,0 +1,30 @@
+import getConfig from "next/config";
```

Comment:
Could you move this back to where it was?
Its not really a page, so I think its best to keep it outside of `pages`